.github/workflows/gradle.yml

```diff
@@ -77,3 +77,13 @@ jobs:
           SLACK_COLOR: DC143C
           SLACK_TITLE: "Build failure"
           SLACK_FOOTER: ""
+
+      - name: Slack Notification - Success
```

Comment:
Slack actions don't support conditionals on env variables so we duplicate this block